app/views/works/_filters.html.erb

```diff
@@ -162,6 +162,28 @@
       <dd class="submit actions"><%= f.submit ts("Sort and Filter") %></dd>
     </dl>
 
+    <% if @collection %>
+    <p class="footnote">
+      <%= link_to ts("Clear Filters"), original_path(@collection) %>
+    </p>
+    <% elsif @tag %>
+    <p class="footnote">
+      <%= link_to ts("Clear Filters"), original_path(@tag) %>
+    </p>
+    <% elsif @fandom %>
+    <p class="footnote">
+      <%= link_to ts("Clear Filters"), original_path(@fandom) %>
+    </p>
```

Comment:
In the `WorksController`, the `@fandom` variable is only used in conjunction with another owner. In theory it could be used with any of the other four owners (`@tag`, `@collection`, `@pseud`, `@user`), but in practice there's only one link with `fandom_id` set:
https://github.com/otwcode/otwarchive/blob/59bb5ebc3ca4fa86056b410d922152905b8cc179/app/views/users/_contents.html.erb#L13-L17
So `fandom_id` needs to be passed as a param for at least the `@user` and `@pseud` "Clear Filters" links, and you should probably avoid having a separate branch for `@fandom`.

(You might also want to think about using the `@owner` variable to simplify your code: https://github.com/otwcode/otwarchive/blob/59bb5ebc3ca4fa86056b410d922152905b8cc179/app/controllers/works_controller.rb#L770
Both here and for the bookmark filters.)